src/Symfony/Component/Routing/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -111,8 +120,8 @@ public function testLoadWithResource()
 
     public function testLoadRouteWithControllerAttribute()
     {
-        $loader = new YamlFileLoader(new FileLocator(array(__DIR__.'/../Fixtures/controller')));
-        $routeCollection = $loader->load('routing.yml');
+        $this->loader = new YamlFileLoader(new FileLocator([__DIR__ . '/../Fixtures/controller']));
```

Comment:
why changing this to a class property everywhere ? The local variable is totally OK (and even avoids the need to look for other places using it).

Thus, it makes merging branches a pain.

Reply:
I agree, this should be reverted